Sources/Tracking/Background Queue/Type/QueueTaskRunResults.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public struct QueueTaskRunResults: Codable, AutoLenses, Equatable {
+    let totalRuns: Int
```

Comment:
this'd be a good place to keep debug info like `lastError`

Reply:
I thought of that but wasn't sure on the best way to store an `Error` object. I could get the `error.localizedDescription` but just a description of an error hides details (like a stacktrace). So because we are missing some useful data, I wasn't sure if it was worth storing this information? What do you think?